packages/base/src/util/AriaLabelHelper.ts

```diff
@@ -57,7 +103,105 @@ const getAssociatedLabelForTexts = (el: HTMLElement) => {
 	return undefined;
 };
 
+const _createInvalidationCallback = (el: UI5Element, trackedProperties: string[]) => {
+	const invalidationCallback = (param: any) => {
```

Comment:
`param: any` => `changeInfo: ChangeInfo`, export `ChangeInfo` from `UI5Element.ts` and import it here for usage. This is ok since we're using it from the `base` project.